shared/react-native/android/app/src/main/AndroidManifest.xml

```diff
@@ -12,7 +12,8 @@
     <uses-permission android:name="${applicationId}.permission.C2D_MESSAGE" />
     <uses-permission android:name="android.permission.VIBRATE" />
     <uses-permission android:name="android.permission.RECEIVE_BOOT_COMPLETED"/>
-
+    <uses-permission android:name="android.permission.READ_CONTACTS" />
+    <uses-permission android:name="android.permission.READ_PROFILE" />
```

Comment:
do we actually need read_profile?

Reply:
Hmm I tested again without it, and didn't get the crasher. Maybe it was unrelated, I removed the permission.